src/crypto.ts

```diff
@@ -14,6 +14,9 @@ const FULFILLMENT_GENERATION_STRING = Buffer.from('ilp_stream_fulfillment', 'utf
 const TOKEN_LENGTH = 18
 const SHARED_SECRET_GENERATION_STRING = Buffer.from('ilp_stream_shared_secret', 'utf8')
 
+const fulfillmentKeyMap = new Map()
+const pskKeyMap = new Map()
```

Comment:
This does not seem like a good solution for speeding up crypto operations. Either these keys should be stored on the connection object so they're automatically cleaned up when it's closed, or we should just make sure these operations are fast. Hmac-ing should basically be free so if the way this is getting webpacked and loaded in the browser makes it use a pure js version we should make sure it uses Web Crypto instead when it's in the browser.

Reply:
No matter how fast hmac is the persisting the keys will be faster than regeneration every time. Ill look into what webpack is using since that would still be beneficial to speed up the fulfillment hmac which occurs on every packet.